src/schemas/state.yaml

```diff
@@ -54,3 +54,24 @@ StorageProof:
       type: array
       items:
         $ref: '#/components/schemas/bytes'
+Account:
+  title: Account
+  type: object
+  required:
```

Comment:
Propose including the blockHash this was resolved from to the return value to support the case that a tag was the block param passed in. If the EL is providing the account state as of the "latest" block, the caller likely has no idea what block that is. This api is not in the `engine` API so we can't assume the caller is aware of current chain state.

Reply:
I think we already have this issue with all the other state reading methods, such as `eth_getBalance`, `eth_getTransactionCount`, etc. I think it is better to retain consistency with the other methods and in the future possible move them all to this format above.

Reply:
Yeah, I agree with @lightclient